src/services/rollouts.js

```diff
@@ -124,6 +125,98 @@ module.exports = class RolloutsHelper {
 			throw error
 		}
 	}
+	/**
+	 * Rollout details
+	 * @method
+	 * @name details
+	 * @param {String} rolloutId - Rollout id
+	 * @param {String} organization_id - Organization id
+	 * @returns {JSON} - Rollout Details
+	 */
+	static async details(rolloutId, orgId) {
+		try {
+			let result = {
+				organization: {},
+			}
+
+			const rollout = await rolloutQueries.findOne({
+				id: rolloutId,
+				organization_id: orgId,
+				type: common.ROLLOUT_TYPE_PROGRAM,
+			})
+
+			if (!rollout) {
+				return responses.failureResponse({
+					message: 'ROLLOUT_NOT_FOUND',
+					statusCode: httpStatusCode.bad_request,
+					responseCode: 'CLIENT_ERROR',
+				})
+			}
+
+			//get the data from storage
+			if (rollout.blob_path) {
+				const response = await filesService.fetchJsonFromCloud(rollout.blob_path)
+				if (
+					response.statusCode === httpStatusCode.ok &&
+					response.result &&
+					Object.keys(response.result).length > 0
+				) {
+					let resultData = response.result
```

Comment:
@adithyadinesh0412  can u check with frontend team ? did they want the entity name in response?

Reply:
Checked with FE team, They are Ok with only id in entity_targeting. @priyanka-TL